src/plugins/numberline/numberline-tile.tsx

```diff
@@ -20,22 +20,28 @@ export const NumberlineTile: React.FC<ITileProps> = observer(function Numberline
 
   //---------------- Calculate Width Of Tile / Scale ----------------------------------------------
   const documentScrollerRef = useRef<HTMLDivElement>(null);
+
   const [tileWidth, setTileWidth] = useState(0);
   const containerWidth = (tileWidth * kContainerWidth);
-  const axisWidth = (tileWidth * kAxisWidth);
+  const axisWidth = tileWidth * kAxisWidth;
   const xShiftNum = ((containerWidth - axisWidth)/2);
   const numToPx = (num: number) => num.toFixed(2) + "px";
   const xScale = createXScale(axisWidth);
 
+
   useEffect(() => {
     let obs: ResizeObserver;
     if (documentScrollerRef.current) {
       obs = new ResizeObserver(() => {
-        setTileWidth(documentScrollerRef.current?.clientWidth ?? 0);
+        if (documentScrollerRef.current?.clientWidth){
+          const newTileWidth  = documentScrollerRef.current?.clientWidth;
+          setTileWidth(newTileWidth ?? 0);
+        }
       });
       obs.observe(documentScrollerRef.current);
     }
     return () => obs?.disconnect();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
It looks like this is disabling line is not required, it is causing a lint warning.